app/src/main/res/layout/activity_toolbar_search.xml

```diff
@@ -1,32 +1,140 @@
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/home_view"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:theme="@style/AppTheme">
+                                        xmlns:app="http://schemas.android.com/apk/res-auto"
+                                        xmlns:tools="http://schemas.android.com/tools"
+                                        android:id="@+id/home_view"
+                                        android:layout_width="match_parent"
+                                        android:layout_height="match_parent"
+                                        android:theme="@style/AppTheme">
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+        <Button
+            android:id="@+id/route_indicator"
+            android:layout_width="match_parent" android:layout_height="60dp"
+            android:layout_marginLeft="14dp" android:layout_marginTop="22dp"
+            android:layout_marginRight="14dp"
+            android:backgroundTint="@color/background" android:text="Button"
+            android:textAllCaps="false" android:visibility="gone"/>
+        <android.support.constraint.ConstraintLayout
+            android:id="@+id/route_switcher_layout"
+            android:layout_width="match_parent" android:layout_height="match_parent"
+            android:background="@color/background" android:visibility="gone">
+
+            <LinearLayout
+                android:id="@+id/route_switcher_menu"
+                android:layout_width="336dp"
+                android:layout_height="95dp" android:layout_marginStart="36dp"
+                android:layout_marginTop="24dp"
+                android:layout_marginEnd="36dp"
+                android:layout_marginBottom="8dp"
+                android:background="@drawable/rounded_bg"
+                android:backgroundTint="@color/background"
+                android:elevation="8dp"
+                android:orientation="horizontal"
+                app:layout_constraintBottom_toTopOf="@+id/route_switcher_list"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+                <LinearLayout
```

Comment:
Ideally we should have a line break in between each separate component, but it isn't really a big deal